frontend/poems/display-poems.js

```diff
@@ -14,18 +14,32 @@ function getPoems () {
   const req = { method: 'GET' }
   fetch('poems.json', req)
     .then(res => res.json())
-    .then(data => appendData(data))
+    .then(data => displayData(data))
     .catch(err => console.log('error: ' + err))
 }
 
-function appendData (data) {
-  const mainCont = document.querySelector('#poem')
-
-  for (let i = 0; i < data.length; i++) {
-    const div = document.createElement('div')
-    div.innerHTML = `${data[i].text} ...signed ${data[i].author} --
-                        ${data[i].location} -- ${data[i].datetime}
-                        -- ROUTE ${data[i].route}`
-    mainCont.appendChild(div)
+function displayData (data) {
+  
+  
+  var i = 0
+  function myLoop(){
+    let rand = Math.floor(Math.random() * data.length)
+    let text = `<b>${data[rand].poem}</b> 
```

Comment:
ur test data looks like this:
```json
{
        "poem": "hello",
        "date": 145999,
        "author": "poe",
        "lat": 41.881832,
        "long": -87.623177,
        "route": 75
      }
```
but the actual poems in the database look like this:
```json
{
    "datetime": 1669842423696,
    "location": [
      41.78889190252645,
      -87.59912079718956
    ],
    "text": "this is a poem example",
    "author": "nick",
    "route": "171"
  }
```

notice that "peom" is called "text", also "date" is called "datetime" && there are no "lat" or "long" properties, instead there's a "location" property which is an array w/the two coordinates in it.